modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectField.tsx

```diff
@@ -43,27 +44,48 @@ const headers = new Headers({
 	'Content-Type': 'application/json',
 });
 
-const ModalAddObjectField: React.FC<IProps> = ({apiURL, observer, onClose}) => {
+const ModalAddObjectField: React.FC<IProps> = ({
+	apiURL,
+	ffObjectFieldBusinessTypeConfigurationEnabled,
+	objectFieldBusinessTypes,
+	observer,
+	onClose,
+}) => {
 	const [error, setError] = useState<string>('');
 	const [picklist, setPicklist] = useState<TPicklist[]>([]);
 	const initialValues: TInitialValues = {
+		businessType: '',
+		dbType: '',
 		label: '',
 		listTypeDefinitionId: 0,
 		name: undefined,
 		required: false,
-		type: '',
+	};
+
+	const getDBType = (businessType: string) => {
```

Comment:
You can reuse this function and return a selectObjectFieldType which contains businessType and dbType, using a [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to take the first array item. Like this:

```ts	
const getSelectObjectFieldType = (businessType: string) => {
    const filteredObjectFieldType = objectFieldBusinessTypes.filter(
	(objectFieldType) => objectFieldType.businessType === businessType
    );

    const [selectObjectFieldType] = filteredObjectFieldType;

    return selectObjectFieldType;
};
```